### DISCUSSION TITLE
New neutorna wand from haslab

### DISCUSSION THREAD

**Discussion by ecto1a2003** [gpstar81/GPStar-proton-pack#239] [General]
https://hasbropulse.com/products/ghostbusters-plasma-series-neutrona-wand-1984

Seems to have some wiring updates?

**Comment by gpstar81** [comment]
I think they were referring to the wiring update from the V2 Spengler wand for the barrel not snapping the wires?

**Comment by ecto1a2003** [comment]
Guess we wont know for sure till they're in hand, i can volunteer mine for teardown visuals if needed.

**Comment by nomakewan** [comment]
![IMG_6946s](https://github.com/user-attachments/assets/9f69ce20-d6a2-48f1-99c0-6c157e2b471f)

Just to give this discussion closure, the 1984 wands are what I would consider a "v3" wand; they, and all Spengler wands released after the 1984 wand came out, have significantly better wiring than the v1 and v2 wands. Rather than glue and tape and a dream, the barrel wire is held by an actual wire relief, and the gap for the barrel wing button wire is narrower to hold the wire in place better. This makes for a significantly cleaner interior and pretty much zero chance of your barrel wires ever breaking due to the pop mechanism.
